README.md

```diff
@@ -324,26 +324,6 @@ Class | Key | Type | Description
 ------|-----|------|------------
 SidelineSpout | start-sideline | Counter | How many `Start Sideline` requests have been received.
 SidelineSpout | stop-sideline | Counter | How many `Stop Sideline` requests have been received.
-VirtualSpout | `virtual-spout-id`.emit | Counter | Tuple emit count per VirtualSpout instance.
```

Comment:
These would need to be auto-generated.

Reply:
Let's revisit this part after your doc task PR.